snippets/language-html.cson

```diff
@@ -1,4 +1,4 @@
-'.text.html, .text.html .meta.scope.between-tag-pair, .text.html .punctuation.tag.begin':
+'.text.html, .text.html .meta.scope.between-tag-pair, .text.html .punctuation.tag.begin, .source.gfm':
```

Comment:
Such as? Any HTML syntax should be valid in Markdown files, that's not the case in GFM?

This is merely a proposal, only to bring up a discussion. And will it override the snippets of language-gfm like "code", "img"? If it will, I'll drop this change.

This comes from my personal uses that I sometimes need to use vanilla HTML when editing Markdown files coz the syntax only covers a small set of HTML elements, then I just wish the snippets of language-html are available.